mock/cpp_mock_scenarios/src/follow_front_entity/decelerate_and_follow.cpp

```diff
@@ -30,10 +29,7 @@ class DecelerateAndFollowScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit DecelerateAndFollowScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "decelerate_and_follow",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.